packages/email/email.js

```diff
@@ -228,8 +240,9 @@ Email.send = function (options) {
     customTransport({ packageSettings, ...options });
     return;
   }
-  if (Meteor.isProduction || process.env.MAIL_URL || Meteor.settings.packages?.email) {
+  if (Meteor.isProduction && process.env.MAIL_URL || Meteor.settings.packages?.email) {
```

Comment:
I think you want to test here if it's production AND if there is an email URL, correct? If yes, you're missing some parenthesis:

```
if (Meteor.isProduction && (process.env.MAIL_URL || Meteor.settings.packages?.email)) {
```

&& has precedence over ||

Reply:
That is something we need to decide. Both the fixed and your proposed conditions are valid options.

Reply:
So you're saying that if it's a development or staging environment but the `Meteor.settings.packages.email` is set, this code should run?

I don't think both conditions are valid because one says that this code should run in production if an email URL is set, as the other says that you can use settings to overwrite a behavior that should occur just in production.

Can you tell what the purpose of this code is? 

I'm not sure which one is correct. I will try to understand this code to see what it's supposed to happen there.